.flowcoverage-tests-threshold

```diff
@@ -1,5 +1,8 @@
 {
   "concurrentFiles": 1,
+  "globExcludePatterns": [
+    "src/ProgramChangeController.test.js"
```

Comment:
Were there issues typing material in this file?  Do we have an option to exclude those parts, perhaps using `$FlowIgnore`?